### PR TITLE
Add Vulnerability Dashboard and Launch API configs

### PR DESCRIPTION
**Pull request by gmcculloug**
Added:
[RHCLOUD-27177](https://issues.redhat.com/browse/RHCLOUD-27177) - Vulnerability Dashboard
[RHCLOUD-27190](https://issues.redhat.com/browse/RHCLOUD-27190) - Launch

Fixed:
[RHCLOUD-27178](https://issues.redhat.com/browse/RHCLOUD-27178) - Remove OpenShift Tag for Vulnerability Management

### CODE REVIEW

File: packages/discovery/Discovery.yml [hunk 1]
```diff
@@ -92,6 +92,15 @@ apis:
         apiType: openapi-v3
         tags:
           - integrations-and-notifications
+      - id: launch
```
Comment (josejulio) [inline packages/discovery/Discovery.yml:95]: This one is missing the `url` property.

Comment (gmcculloug) [reply packages/discovery/Discovery.yml:95]: 🤦 I'll get that fixed in a little bit, thanks for catching that.

Comment (gmcculloug) [reply packages/discovery/Discovery.yml:95]: @josejulio done

**Comment by josejulio** [comment]
It looks like there is one validation failing regarding the length of the description. I'll update that on Monday.

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**